src/redux/actions/moviesAction.ts

```diff
@@ -0,0 +1,57 @@
+import { AppDispatch } from "../../types/redux.interface";
+import Movie from "../../types/movie.interface";
+import { GET_MOVIES_ERROR, GET_MOVIES_START, GET_MOVIES_SUCCESS } from "../types";
+
+const BACKEND_URL = "http://localhost:4000/movies";
+const MOVIES_PER_PAGE = "9";
+const SORT_ORDER = "desc";
+
+export function getAllMoviesAction(numPage: number, sortBy: string) {
+    return async(dispatch: AppDispatch) => {
+        dispatch(getAllMoviesStart());
+
+        try {
+            const searchParams = new URLSearchParams({ limit: MOVIES_PER_PAGE, offset: numPage.toString(), sortBy, sortOrder: SORT_ORDER });
```

Comment:
I'm not sure that you need to use `toString()`:
```suggestion
            const searchParams = new URLSearchParams({ limit: MOVIES_PER_PAGE, offset: numPage, sortBy, sortOrder: SORT_ORDER });
```

Reply:
Resolved